### ISSUE TITLE
Armor Texture different

### ISSUE THREAD

**Issue by krwminer** [DivineRPG/DivineRPG#41]
When I put on a set of now the proper blue rupee armor I still appear to have on a purple armor when using F5 or to other players.

**Comment by Eternaldoom** [comment]
Rupee armor has always been purple, but now that you mention it I suppose I'll change it

**Comment by krwminer** [comment]
Rupee Tools and Armor pieces have always been blue. The displayed armor set has always been purple and looked nothing like the armor items. It was reported as a bug way way back and it was on the list to be fixed at some point.

I looked at your recent change to the corrupted armor and misread it. I thought you were changing the armor from the blue to purple to match the armor item pieces/tools/ingots, but you changed it from purple to blue.

In summary, I think to fix a long standing bug, I would make the rupee armor set a deep blue like the armor item pieces and the corrupted armor set a purple like the armor pieces. That makes the most sense to me.

**Comment by Eternaldoom** [comment]
I will. I'll also properly do multicolored rupee.

**Comment by Eternaldoom** [comment]
I used the old blue rupee texture as default rupee (since its closer to the color of a rupee ingot), made a new default rupee texture, and readded the colored rupee armor. Blue rupee item textures are the old default rupee.
